src/main/java/com/microsoft/sqlserver/jdbc/Geography.java

```diff
@@ -15,7 +15,13 @@
     
     /**
      * Private constructor used for creating a Geography object from WKT and srid.
+     * 
+     * @param WellKnownText
+     *              Well-Known Text (WKT) provided by the user.
+     * @param srid 
+     *             Spatial Reference Identifier (SRID) provided by the user.
      * @throws SQLServerException 
+     *              if an exception occurs
      */
     private Geography(String WellKnownText, int srid) throws SQLServerException {
```

Comment:
seems a bit inconsistent - why is this spelled out "WellKnownText" when the others are all wkt?